### ISSUE TITLE
Duplicate exercise names

### ISSUE THREAD

**Issue by miku86** [Kaltsoon/react-native-material#16]
Hey folks,

In part [2-react-native-basics.md](https://github.com/Kaltsoon/react-native-material/blob/master/material/2-react-native-basics.md), there are multiple exercises with the same name:

- [Exercise 10.6.](https://github.com/Kaltsoon/react-native-material/blob/master/material/2-react-native-basics.md#exercise-106)
- [Exercise 10.6.](https://github.com/Kaltsoon/react-native-material/blob/master/material/2-react-native-basics.md#exercise-106-1)
- [Exercise 10.7.](https://github.com/Kaltsoon/react-native-material/blob/master/material/2-react-native-basics.md#exercise-107)
- [Exercise 10.7.](https://github.com/Kaltsoon/react-native-material/blob/master/material/2-react-native-basics.md#exercise-107-1)

I don't know if this is on purpose, so I'll let you update it on your own.

**Comment by Kaltsoon** [comment]
Yes, we will fix these once we have locked the number of exercises. Because the numbers are added manually, adding an exercise requires to change every number after the exercise. This is a bit tedious.
